Automattic/wp-calypso
# Add screens to show when a user tries to visit a private site

This pull request adds screens for private sites to Calypso. They aren't yet functional, this will follow in another PR.

#### Testing instructions
  
1. Run `git checkout add/private-site` and start your server
2. Open the [`Private site` page](http://calypso.localhost:3000/log-in/private-site) in an incognito window
3. Check that you see a notice telling you that this site is private, and a log in form:
<img width="515" alt="screen shot 2017-07-26 at 18 02 29" src="https://user-images.githubusercontent.com/275961/28633849-0ad2860a-722d-11e7-8179-46f882abb770.png">

2. Open the [`Private site` page](http://calypso.localhost:3000/log-in/private-site), when you are logged in to WordPress.com
3. Check that you see a page which gives you the option to request access to the private site:
<img width="508" alt="screen shot 2017-07-26 at 18 02 19" src="https://user-images.githubusercontent.com/275961/28633850-0ad81070-722d-11e7-9215-f3105ce6fea4.png">

#### Reviews
